README.md

```diff
@@ -0,0 +1,26 @@
+# @homebound/activesupport
+
+This package provides prototype extensions, similar to the Ruby
+[ActiveSupport](https://guides.rubyonrails.org/active_support_core_extensions.html) project.
+
+## Installation
+
+```bash
+npm install --save @homebound/activesupport
+```
+
+```bash
+yarn add @homebound/activesupport
+```
+
+## Usage
+
+Simply import the package to add the extensions to the prototypes.
+
+```typescript
+import "@homebound/activesupport";
+```
+
+## Documentation
+
+For now, check out [the source](/src/).
```

Comment:
Usually, I'd use something like `tsdoc` to auto-generate this. However, since these are prototype extensions, the `tsdoc` is sparse.

We should think about the best documentation strategy moving forward.